pipelines/smartseq2_multisample/README.md

```diff
@@ -4,22 +4,24 @@
 
 # Smart-seq2 Multi Sample Pipeline
 ## Introduction
-The Smart-seq2 Multi Sample pipeline is a wrapper around the [Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) pipeline. It is developed by the Data Coordination Platform of the Human Cell Atlas to process single-cell RNAseq (scRNAseq) data generated by Smart-seq2 assays. The workflow processes multiple cells by importing and running the [Smart-seq2 Single Sample workflow](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) for each individual cell and then merging the resulting Zarr output files and optionally generating a Loom file. 
+The Smart-seq2 Multi Sample pipeline is a wrapper around the [Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) pipeline. It is developed by the Data Coordination Platform of the Human Cell Atlas to process single-cell RNAseq (scRNAseq) data generated by Smart-seq2 assays. The workflow processes multiple cells by importing and running the [Smart-seq2 Single Sample workflow](/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) for each individual cell and then merging the resulting Zarr output files and optionally generating a Loom file. 
```

Comment:
Change `[Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample)` to use a relative path too, unless there's a reason that it shouldn't be changed.

Reply:
Changed.